internal/consensus/metrics.go

```diff
@@ -125,11 +125,17 @@ type Metrics struct {
 	// in.
 	LateVotes metrics.Counter `metrics_labels:"vote_type"`
 
-	// ProposalTimestampDifference is the difference between the timestamp in
-	// the proposal message and the local time of the validator at the time
-	// that the validator received the message.
-	// metrics:Difference between the timestamp in the proposal message and the local time of the validator at the time it received the message.
-	ProposalTimestampDifference metrics.Histogram `metrics_bucketsizes:"-10, -.5, -.025, 0, .1, .5, 1, 1.5, 2, 10" metrics_labels:"is_timely"`
+	// ProposalTimestampDifference is the difference between the local time
+	// of the validator at the time it receives a proposal message, and the
+	// timestamp of the received proposal message.
+	//
+	// The buckets are defined based on the default values of the Synchrony
+	// consensus parameters: Precision = 0.5s and MessageDelay = 2.0s. The
+	// first bucket represents differences less than than -Precision, while
+	// the last twobuckets represent differences greater than (MessageDelay
+	// + Precision). In both extremes, the proposal is not timely.
+	// metrics:Difference between the local time when a proposal message is received and the timestamp in the proposal message.
+	ProposalTimestampDifference metrics.Histogram `metrics_bucketsizes:"-.5, 0, .5, 1, 1.5, 2, 2.5, 3" metrics_labels:"is_timely"`
```

Comment:
Is there a link to a discussion/issue where there's consensus that these values should replace the old ones?

Reply:
There's an issue for that, I think. And also we agreed yesterday that the QA folks will run a preliminary 200-node run to gather metrics and fine-tune these values

Reply:
https://github.com/cometbft/cometbft/issues/2323

We should add there this requirement.

Reply:
Added.